### ISSUE TITLE
Conan recipe not updated for Conan 2.0.0

### ISSUE THREAD

**Issue by adityasivarajmz** [foxglove/ws-protocol#380]
**Description**
I had asked above the Conan version clash issue yesterday ([here](https://github.com/foxglove/ws-protocol/issues/351#issuecomment-1445946950)). Fortunately, a fix was already in the works and it was published a few hours later. 

But, now when I am trying to build, I am getting this error:

```
foxglove-websocket/1.0.0: RUN: cmake -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="/home/minuszero-ams/.conan2/p/t/foxglbb25eebe40dcb/b/build/Release/generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="/home/minuszero-ams/.conan2/p/t/foxglbb25eebe40dcb/p" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Release" "/home/minuszero-ams/.conan2/p/t/foxglbb25eebe40dcb/b/."
CMake Error: The source directory "/home/minuszero-ams/.conan2/p/t/foxglbb25eebe40dcb/b" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.

foxglove-websocket/1.0.0: ERROR: 
Package '843428e8a9b40ca5d55e858550cbf6dc3e0054c4' build failed
foxglove-websocket/1.0.0: WARN: Build folder /home/minuszero-ams/.conan2/p/t/foxglbb25eebe40dcb/b/build/Release
*********************************************************
Recipe 'foxglove-websocket/1.0.0' cannot build its binary
It is possible that this recipe is not Conan 2.0 ready
If the recipe comes from ConanCenter check: https://conan.io/cci-v2.html
If it is your recipe, check it is updated to 2.0
*********************************************************

ERROR: foxglove-websocket/1.0.0: Error in build() method, line 38
	cmake.configure()
	ConanException: Error 1 while executing
```
I am not sure if this is something that can be fixed from your side, or from Conan's. If it must be fixed by Conan, please close this issue. But if there is an issue with my method, please let me know.

- Version:
- Platform: Ubuntu 20.04

**Steps To Reproduce**

```
cd cpp
mkdir -p src/foxglove-websocket
cp ../foxglove-websocket/conanfile.py foxglove-websocket/conanfile.py
conan install foxglove-websocket -s compiler.cppstd=17 --build=missing
cp -r ../foxglove-websocket/ foxglove-websocket/
conan create foxglove-websocket -s compiler.cppstd=17
```

**Expected Behavior**
The command completes without any errors.

**Comment by achim-k** [comment]
foxglove-websocket/1.0.0 has not yet been published to conan-center. We are working on it.

**Comment by jtbandes** [comment]
Is this still an issue? Looks like 1.0.0 is published but 1.1.0 is not yet (https://github.com/conan-io/conan-center-index/pull/18885)?

**Comment by defunctzombie** [comment]
https://github.com/conan-io/conan-center-index/pull/18885 has merged. If you are still unable to install via conan v2 please let us know any new issues.
